arangod/Graph/GraphManager.h

```diff
@@ -93,12 +93,13 @@ class GraphManager {
   /// @brief find or create collections by EdgeDefinitions
   ////////////////////////////////////////////////////////////////////////////////
   OperationResult findOrCreateCollectionsByEdgeDefinitions(
-      std::map<std::string, EdgeDefinition> const& edgeDefinitions,
+      Graph const& graph, std::map<std::string, EdgeDefinition> const& edgeDefinitions,
       bool waitForSync, VPackSlice options);
 
-  OperationResult findOrCreateCollectionsByEdgeDefinition(EdgeDefinition const& edgeDefinition,
+  OperationResult findOrCreateCollectionsByEdgeDefinition(Graph const& graph,
+                                                          EdgeDefinition const& edgeDefinition,
                                                           bool waitForSync,
-                                                          VPackSlice options);
+                                                          VPackSlice const options);
```

Comment:
```suggestion
                                                          VPackSlice options);
```
Adding `const` at the function declaration does make no difference, as the parameter is passed by value and thus the function cannot modify the original value anyway. It's good to leave it at the definition, though, where it prohibits to modify the local variable.